extensions/binding/org.eclipse.smarthome.binding.tradfri/README.md

```diff
@@ -0,0 +1,58 @@
+# Trådfri Binding
+
+This binding integrates the IKEA Trådfri gateway and devices connected to it (such as dimmable LED bulbs).
+
+## Supported Things
+
+Besides the gateway (thing type "gateway"), the binding currently supports dimmable warm white bulbs as well as white spectrum bulbs.
+
+The thing type ids are defined according to the lighting devices defined for ZigBee LightLink ([see page 24, table 2](https://www.nxp.com/documents/user_manual/JN-UG-3091.pdf). These are:
+
+| Device type              | ZigBee Device ID | Thing type |
+|--------------------------|------------------|------------|
+| Dimmable Light           | 0x0100           | 0100       |
+| Colour Temperature Light | 0x0220           | 0220       |
+
+
+## Thing Configuration
+
+The gateway requires a `host` parameter for the hostname or IP address and a `code`, which is the security code that is printed on the bottom of the gateway. Optionally, a `port` can be configured, but any standard gateway uses the default port 5684.
+
+The devices require only a single parameter, which is their instance id. Unfortunately, this is not displayed anywhere in the IKEA app, but it seems that they are sequentially numbered starting with 65537 for the first device. If in doubt, use the auto-discovered things to find out the correct instance ids.
+
+## Channels
+
+All devices support the `brightness` channel, while the white spectrum bulbs additionally also support the `colorTemperature` channel.
+
+| Channel Type ID | Item Type     | Description                           |
```

Comment:
You would make me happy by formatting the tables in the MD file as well :-D

http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table 👍